src/lib/components/SteppedList/SteppedList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SteppedList.css';
+
+const SteppedList = (props) => {
+  if (props.children) {
+    const steppedListItems = React.Children.map(props.children,
+      (child, index) => React.cloneElement(child, {
+        index,
+        detailed: props.detailed,
+      }),
+    );
+
+    return (
+      <ol className={props.detailed ? 'p-stepped-list--detailed' : 'p-list-step'}>
+        {steppedListItems}
+      </ol>
+    );
+  }
+
+  return null;
+};
+
+SteppedList.defaultProps = {
+  children: '',
+  detailed: false,
+};
+
+SteppedList.propTypes = {
+  children: PropTypes.node,
```

Comment:
I think it would be cleaner to have an array: `PropTypes.array` instead of a node. Because here it meens that you can pass absolutely anything that can be rendered. https://reactjs.org/docs/typechecking-with-proptypes.html

Maybe it is the behaviour you want?

Reply:
This would give an error if only one child is present though?

Reply:
This would give an error if only one child is present though? I think PropTypes.node is fine because it allows single/multiple React elements (SteppedListItem), and also basic li tags if someone wanted to get really granular with customisation